app/decorators/physical_chassis_decorator.rb

```diff
@@ -0,0 +1,24 @@
+class PhysicalChassisDecorator < MiqDecorator
+  def self.fonticon
+    'pficon pficon-cluster'
+  end
+
+  def fileicon
+    "svg/vendor-#{vendor.downcase}.svg"
+  end
+
+  def quadicon
+    {
+      :top_left     => {
+        :text    => t = physical_servers.count,
+        :tooltip => n_("%{number} Physical Servers", t) % {:number => t}
```

Comment:
You need three arguments here: singular form, plural form and the number itself:
```ruby
:tooltip => n_("%{number} Physical Server", "%{number} Physical Servers", t) % {:number => t}
```